src/main/java/com/commercetools/sync/commons/models/WaitingToBeResolvedCategories.java

```diff
@@ -0,0 +1,35 @@
+package com.commercetools.sync.commons.models;
+
+import io.sphere.sdk.categories.CategoryDraft;
+
+import javax.annotation.Nonnull;
+import java.util.Set;
+
+public final class WaitingToBeResolvedCategories<T extends CategoryDraft> extends WaitingToBeResolved<T> {
+    private T waitingDraft;
+
+    // Needed for the 'com.fasterxml.jackson' deserialization, for example, when fetching
+    // from CTP custom objects.
+    public WaitingToBeResolvedCategories() {
```

Comment:
i think you don't need this constructor

Reply:
I need it please read the comment from above